scripts/python/readPly.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
```

Comment:
pls decribe what is this script for
is this needed by the current pixi shell build or can be put in a different PR?